OpenImis.RestApi/Startup.cs

```diff
@@ -32,10 +30,13 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            // Add the DbContext 
-            //services.AddDbContext<IMISContext>(options => options.UseSqlServer(Configuration.GetConnectionString("IMISDatabase")));
+            var configImisModules = Configuration.GetSection("ImisModules").Get<List<ConfigImisModules>>();
```

Comment:
Should we sort the list by Version field?

Reply:
This is not necessary here. The next line of code already get the version